sections/semantics-embedded-content.include

```diff
@@ -450,7 +450,7 @@ was an English &lt;a href="/wiki/Music_hall"&gt;music hall&lt;/a&gt; singer, ...
     <dd><a>Global aria-* attributes</a></dd>
     <dt><a>DOM interface</a>:</dt>
     <dd>
-      <pre class="idl" data-highlight="webidl" dfn-for="HTMLSourceElement">
+      <pre class="idl" data-highlight="webidl" dfn-for="HTMLSourceElement" id='Picture-HTMLSourceElement'>
```

Comment:
Should not define an ID here there is no need to define this since HTMLSourceElement is already unique. You would want to change the dfn-for attribute to something like HTMLPictureElement/HTMLSourceElement. But even this is weird. Partials don't have a good solutions when they exist in the same document.